### DISCUSSION TITLE
Error when I use Create Ticket button

### DISCUSSION THREAD

**Discussion by DroidBV8** [glpi-project/glpi-agentmonitor#8] [Q&A] [answered]
Hello, 

I have installed GLPI agent monitor via GLPI agent 1.5 with AGENTMONITOR=1 argument

Install is fine but I have this error when I click on Create ticket :

![image](https://github.com/glpi-project/glpi-agentmonitor/assets/71468154/cbb844ea-c4e4-48b0-8ad8-72f9c4950f2c)

It's a bug ?
An idear ?

Best regards

**Comment by redddcyclone** [comment]
Hi @DroidBV8 

Could you post an screenshot from `regedit.exe` at key `HKEY_LOCAL_MACHINE\SOFTWARE\GLPI-Agent`? Specifically the `server` value.

**Comment by DroidBV8** [comment]
![image](https://github.com/glpi-project/glpi-agentmonitor/assets/71468154/70b361b4-4d77-4fd8-82c8-4a9fcd2959d8)

**Reply by redddcyclone** [reply]
@DroidBV8 

The problem is that you're using quotes on the URLs. Is inventory working OK with this configuration?

**Reply by DroidBV8** [reply]
Yes thats works!!

Here the command I use for installation: 

MsiExec.exe /i "\\XXXX\XXX-XXXX\XXX\glpi_agent\Versions\GLPI-Agent-1.5-x64.msi" /quiet RUNNOW=1 SERVER='https://XXXXX.with20.glpi-network.cloud/marketplace/glpiinventory/' PROXY='http://XXXX.XXXXX-XXXX.fr:3128' ADD_FIREWALL_EXCEPTION=1 AGENTMONITOR=1

Must be 

MsiExec.exe /i "\\XXXX\XXX-XXXX\XXX\glpi_agent\Versions\GLPI-Agent-1.5-x64.msi" /quiet RUNNOW=1 SERVER=https://XXXXX.with20.glpi-network.cloud/marketplace/glpiinventory/ PROXY=http://XXXX.XXXXX-XXXX.fr:3128 ADD_FIREWALL_EXCEPTION=1 AGENTMONITOR=1


?????

**Reply by redddcyclone** [reply, marked as answer]
@DroidBV8 

Yes, for now I think it's better if you remove the quotes like in your example. Anyway, I'll try to check for quotes in the URL retrieve process for the next version. 

Try removing the quotes and please report the results

**Comment by DroidBV8** [comment]
@redddcyclone Definitively better without quotes :)
